app/src/main/java/ar/com/wolox/android/cookbook/room/RoomRecipePresenter.kt

```diff
@@ -27,13 +21,14 @@ class RoomRecipePresenter @Inject constructor(
         } ?: run {
             if (user.isNotEmpty()) {
                 userName = user
-                Thread(Runnable {
-                    val data = db.RoomDataDao().getAll()
-                    handler.post {
-                        view.updateEntities(data)
-                        view.showLoginSuccess()
+
+                GlobalScope.launch(Dispatchers.Main) {
```

Comment:
@mariaAcevedo @FedericoSanucciWX Remember that we should probably never use the `GlobalScope`. Create a `CoroutinesBasePresenter` (copy it from Stayquit or from the Coroutines recipe 😉) and use it.

Reply:
I like it!! and I stole it.... I mean, I took it borrowed from Coroutines recipe :D